crypto-square.json

```diff
@@ -0,0 +1,82 @@
+{
+  "normalized_plaintext": {
+    "description": "the spaces and punctuation are removed from the English text and the message is downcased",
+    "cases" : [
+      {
+        "description": "Lowercase",
+        "plaintext": "Hello",
+        "expected": "hello"
+      },
+      {
+        "description": "Remove spaces",
+        "plaintext": "Hi there",
+        "expected": "hithere"
+      },
+      {
+        "description": "Remove punctuation",
+        "plaintext": "@1, 2%, 3 Go!",
+        "expected": "123go"
+      }
+    ]
+  },
+  "plaintext_segments": {
+"description": "The plaintext should be organized in to a rectangle.  The size of the rectangle (`r x c`) should be decided by the length of the message, such that `c >= r` and `c - r <= 1`, where `c` is the number of columns and `r` is the number of rows.",
+    "cases" : [
+      {
+        "description": "empty plaintext results in an empty rectangle",
+        "plaintext": "",
+        "expected": "[]"
+      },
+      {
+        "description": "4 character plaintext results in an 2x2 rectangle",
+        "plaintext": "Ab Cd",
+        "expected": "[ab, cd]"
```

Comment:
Hmm, this is a single string representing the list. I am wondering whether it would be easier for languages to handle it if it were a list of strings: `["ab", "cd"]`